src/store-api/src/region_request.rs

```diff
@@ -222,7 +222,13 @@ fn make_region_creates(creates: CreateRequests) -> Result<Vec<(RegionId, RegionR
 
 fn parse_region_drop(drop: DropRequest) -> Result<(RegionId, RegionDropRequest)> {
     let region_id = drop.region_id.into();
-    Ok((region_id, RegionDropRequest {}))
+    let fast_drop_database_path = drop.fast_drop_database_path;
```

Comment:
How about using `fast_drop_path`? I believe we might also offer the ability to fast drop physical tables in the future, so it's not just for dropping databases.

Reply:
Done